pandas-stubs/core/indexes/timedeltas.pyi

```diff
@@ -44,6 +44,18 @@ class TimedeltaIndex(DatetimeTimedeltaMixin[Timedelta], TimedeltaIndexProperties
         copy: bool = ...,
         name: str = ...,
     ) -> None: ...
+    def __new__(
```

Comment:
If we don't declare `__init__`, then pyright compares the `__init__` from the parent classes and sees that it is not compatible with `__new__`. Removing `__init__` from `Properties` would allow us to remove `__init__` from `TimedeltaIndex`.